src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -211,9 +226,18 @@ function ReportActionItemMessageEdit(props) {
     const deleteDraft = useCallback(() => {
         debouncedSaveDraft.cancel();
         Report.saveReportActionDraft(props.reportID, props.action.reportActionID, '');
-        ComposerActions.setShouldShowComposeInput(true);
-        ReportActionComposeFocusManager.clear();
-        ReportActionComposeFocusManager.focus();
+
+        if (isActive()) {
+            setShouldShowComposeInputKeyboardAware(true);
+            ReportActionComposeFocusManager.clear();
+            ReportActionComposeFocusManager.focus();
+            if (EmojiPickerAction.isActive(props.action.reportActionID)) {
+                EmojiPickerAction.clearActive();
+            }
+            if (ReportActionContextMenu.isActiveReportAction(props.action.reportActionID)) {
+                ReportActionContextMenu.clearActiveReportAction();
+            }
```

Comment:
Do we need to call `setShouldShowComposeInputKeyboardAware`, `EmojiPickerAction.clearActive` and `ReportActionContextMenu.clearActiveReportAction` here? Won't they get called on the unmount phase?

Reply:
I think we don't need to call them here. Removed